plugins/data_migration/api/data_migration_helper.js

```diff
@@ -492,56 +499,56 @@ module.exports = function(my_db) {
                 }
                 else {
                     if (!res.redirect_url || res.redirect_url === "") {
-                        scripts.push('mongodump ' + dbstr + ' --collection apps -q \'{ _id: ObjectId("' + appid + '") }\' --out ' + my_folder);
+                        scripts.push({cmd: 'mongodump', args: [...dbargs, "--collection", "apps", "-q", '{ _id: ObjectId("' + appid + '") }', "--out", my_folder]});
                     }
                     else {
                         //remove redirect field and add it after dump.
-                        scripts.push('mongo ' + countly_db_name + ' ' + dbstr0 + ' --eval  \'db.apps.update({ _id: ObjectId("' + appid + '") }, { $unset: { redirect_url: 1 } })\'');
-                        scripts.push('mongodump ' + dbstr + ' --collection apps -q \'{ _id: ObjectId("' + appid + '") }\' --out ' + my_folder);
-                        scripts.push('mongo ' + countly_db_name + ' ' + dbstr0 + ' --eval  \'db.apps.update({ _id: ObjectId("' + appid + '") }, { $set: { redirect_url: "' + res.redirect_url + '" } })\'');
+                        scripts.push({cmd: 'mongo', args: [countly_db_name, ...dbargs0, "--eval", 'db.apps.update({ _id: ObjectId("' + appid + '") }, { $unset: { redirect_url: 1 } })']});
+                        scripts.push({cmd: 'mongodump', args: [...dbargs, "--collection", "apps", "-q", '{ _id: ObjectId("' + appid + '") }', "--out", my_folder]});
+                        scripts.push({cmd: 'mongo', args: [countly_db_name, ...dbargs0, "--eval", 'db.apps.update({ _id: ObjectId("' + appid + '") }, { $set: { redirect_url: "' + res.redirect_url + '" } })']});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4577914)